src/realm/util/encrypted_file_mapping.cpp

```diff
@@ -145,8 +145,16 @@ AESCryptor::AESCryptor(const uint8_t* key)
       m_dst_buffer(new char[block_size])
 {
 #if REALM_PLATFORM_APPLE
-    CCCryptorCreate(kCCEncrypt, kCCAlgorithmAES, 0 /* options */, key, kCCKeySizeAES256, 0 /* IV */, &m_encr);
-    CCCryptorCreate(kCCDecrypt, kCCAlgorithmAES, 0 /* options */, key, kCCKeySizeAES256, 0 /* IV */, &m_decr);
+    // Randomize IV
+    void *iv = 0;
+    unsigned char u_iv[kCCKeySizeAES256];
+    for (size_t i = 0; i < sizeof(iv); ++i) {
```

Comment:
This should be `arc4random_buf(u_iv, kCCKeySizeAES256)`.

Reply:
I will fix this, thank you!